lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -197,6 +216,14 @@ function distributeTwoNodes(firstNode, secondNode) {
   return Node.Creator.parenthesis(Node.Creator.operator('+', newArgs));
 }
 
+function hasFraction(args) {
+  return args.filter(isFraction).length > 0;
+}
+
+function isFraction(node) {
```

Comment:
there are some similar functions in `Node.Type`, so you could add it there if you want